darc/darc.go

```diff
@@ -790,6 +793,8 @@ func (s Signer) Identity() Identity {
 		return NewIdentityProxy(s.Proxy)
 	case 4:
 		return NewIdentityEvmContract(s.EvmContract)
+	case 5:
+		return NewIdentityECDSA(s.ECDSA.PublicKey)
```

Comment:
Here and everywhere else: we should call the `ECDSA` identity `MPCECDSA` to indicate we're doing something special..